setup/lib/utils.js

```diff
@@ -0,0 +1,28 @@
+const os = require('os');
+const path = require('path');
+const arch = "amd64";
```

Comment:
There can be other architectures if self-hosted runners are used.

Ref: https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners#architectures

Reply:
Issue created: https://github.com/notaryproject/notation-action/issues/13